### ISSUE TITLE
Support hiding perf counter chart legend

### ISSUE THREAD

**Issue by gildorwang** [merione/public#6]
<a href="https://github.com/gildorwang"><img src="https://avatars.githubusercontent.com/u/1061388?" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [gildorwang](https://github.com/gildorwang)**
_Saturday Feb 16, 2013 at 10:14 GMT_
_Originally opened as https://github.com/gildorwang/meri2/issues/42_

---

May depend on #41

**Comment by RodgeFu** [comment]
won't fix
